docker-jans-shibboleth/scripts/entrypoint.py

```
@@ -0,0 +1,71 @@
+    
+    jetty_home = os.environ.get("JETTY_HOME", "/opt/jetty")
+    jetty_base = JETTY_BASE
+    
+    java_opts = os.environ.get("CN_JAVA_OPTIONS", "-Xms256m -Xmx512m")
+    
+    os.environ["JAVA_OPTIONS"] = java_opts
+    os.environ["JETTY_HOME"] = jetty_home
+    os.environ["JETTY_BASE"] = jetty_base
+    
+    os.chdir(jetty_base)
+    os.execl(
```

Ruff (0.14.14)

[error] Starting a process without a shell

(S606)